.github/workflows/ci.yml

```diff
@@ -107,6 +107,7 @@ jobs:
               run: vendor/bin/phpunit --testsuite=functional --colors=always
               env:
                   DATABASE_URL: mysql://root:root@localhost:3306/contao_test
+                  SYMFONY_DEPRECATIONS_HELPER: disabled=1
```

Comment:
Can you explain this?

Reply:
I knew you would stumble upon it. 😄 

The functional tests currently fail, because of a third-party deprecation that is incorrectly treated as a "self" deprecation:

```
Remaining self deprecation notices (124)

  86x: The "twig.exception_listener" service is deprecated since Symfony 4.4.
    23x in RoutingTest::testResolvesAliases from Contao\CoreBundle\Tests\Functional
    22x in RoutingTest::testResolvesAliasesWithLocale from Contao\CoreBundle\Tests\Functional
    19x in RoutingTest::testResolvesAliasesWithoutUrlSuffix from Contao\CoreBundle\Tests\Functional
    13x in RoutingTest::testResolvesTheRootPageWithLocale from Contao\CoreBundle\Tests\Functional
    8x in RoutingTest::testResolvesTheRootPage from Contao\CoreBundle\Tests\Functional
    1x in RoutingTest::testOrdersThePageModelsByCandidates from Contao\CoreBundle\Tests\Functional

  38x: The "twig.controller.exception" service is deprecated since Symfony 4.4.
    13x in RoutingTest::testResolvesAliases from Contao\CoreBundle\Tests\Functional
    12x in RoutingTest::testResolvesAliasesWithLocale from Contao\CoreBundle\Tests\Functional
    9x in RoutingTest::testResolvesAliasesWithoutUrlSuffix from Contao\CoreBundle\Tests\Functional
    3x in RoutingTest::testResolvesTheRootPageWithLocale from Contao\CoreBundle\Tests\Functional
    1x in RoutingTest::testResolvesTheRootPage from Contao\CoreBundle\Tests\Functional
```

Both the `twig.exception_listener` and the `twig.controller.exception` services use deprecated classes. However, since these services are defined by Symfony and not by Contao, there is no way for us to change them. (I have also tried to overwrite the service definitions, however, then the functional tests no longer work, because other parts of Symfony still expect the old, deprecated class.)

So until this issue has been fixed in Symfony, disabling the deprecation helper for the functional tests seemed the best option. Note that the deprecation helper is still enabled for the unit tests, so most of the deprecations are still reported.

Reply:
Imho it should be configured to `max[self]=0`. Doesn‘t that work?

Reply:
No, because that translates to "fail if there are more than 0 self deprecations". It would work with `max[self]=125`, because we currently have 124 self deprecations. But there is no point in that.

Reply:
Why do we have self deprecations that are not fixed?😱

Reply:
I have explained this in detail above.

Reply:
Okay, I have to check that in more detail but there‘s no way we can merge that like that.

Reply:
Why not? We are already notified about deprecations in the unit tests. No need to run them all again in the functional tests, is there?

Reply:
We can add a `# FIXME` if it makes you feel better. But we really need to merge this PR, because I cannot continue working on Contao 4.9 without it.

Reply:
Yeah I don't want to have this merged because we might forget about it and then stop fixing our deprecations :) 
But again: You state the following

> The functional tests currently fail, because of a third-party deprecation that is incorrectly treated as a "self" deprecation:

Is this a bug that's been worked on or why does that happen? Maybe it's something we can fix?

Reply:
We cannot (in Contao). As I have explained above

> Both the `twig.exception_listener` and the `twig.controller.exception` services use deprecated classes.

The bug is in the Symfony PHPUnit bridge, which considers the deprecated classes "self deprecations", although they are third-party deprecations.

Reply:
Yes, I did understand this. But then we have to report it and add the issue in the FIXME description and not wait for it to randomly vanish?